source/components/call-phone.ts

```diff
@@ -29,3 +34,28 @@ export function callPhone(phoneNumber: string, opts?: Options): void {
 		)
 	}
 }
+
+export const formatNumber = (phoneNumber: string): string => {
+	let re = /^(1|)?(\d{3})(\d{3})(\d{4})$/u
```

Comment:
International numbers can be anything; isn't Great Britain +83?

Reply:
Yes, I agree there are country calling codes other than +1. 

Phone numbers in other countries don't all have exactly ten digits, though, so the formatting here doesn't apply internationally.

Reply:
So we're not looking to support +83 by replacing` ^(1|)` with `^(\d|)` in the regex and `'+1 '` with `'+${match[1]} '` in the international code due to the ten digits.

Reply:
Fair!